vividus-plugin-web-app/src/main/java/org/vividus/ui/web/action/search/CheckboxNameSearch.java

```diff
@@ -77,7 +80,10 @@ private List<WebElement> searchCheckboxByLabels(SearchContext searchContext, Sea
             }
             else
             {
-                checkboxes = label.findElements(getXPathLocator(PRECEDING_SIBLING_CHECKBOX_LOCATOR));
+                checkboxes = Stream
+                        .of(PRECEDING_SIBLING_CHECKBOX_LOCATOR, FOLLOWING_SIBLING_CHECKBOX_LOCATOR,
+                                CHILD_CHECKBOX_LOCATOR)
```

Comment:
do we really need to find checkboxes by all locators? won't the first one be enough?

Reply:
no, because with CHECKBOX_LABEL_FORMAT we find for example child element, but then lost it in this method
![image](https://user-images.githubusercontent.com/15783910/185564245-a01d758d-79ca-4617-8f5c-c1aa02293d06.png)

Reply:
on the other hand, I can change logic and use 'for' cycle and exit once first element collection will be find. What do you think?

Reply:
> exit once first element collection will be find

yes, that I was asking about

Reply:
java.util.stream.Stream#findFirst ??